docs/style-guide.adoc

```diff
@@ -526,3 +523,28 @@ RETURN 1;
 ----
 RETURN 1
 ----
+
+== Recommendations
+
+* When using Cypher language constructs in prose, use a monospaced font and follow the styling rules.
+** When referring to labels and relationship types, include the colon like so: `:Label`, `:REL_TYPE`.
```

Comment:
"When referring to labels and relationship types, include the colon like so:" -> "When referring to node labels and relationship types, the colon should be included as follows:"